docs/develop/customize.md

```diff
@@ -22,3 +24,43 @@ Passing the `--no-install-js` option, skips re-installation of JS dependencies.
 Go to the browser [*https://localhost:5000/*](https://localhost:5000) or refresh the page. And voilà! The logo has been changed!
 
 **WARNING**: If you do not see it changing, check in an incognito window, the browser might have cached the logo.
+
+## Change colors
+
+You might also been wondering: *How do I change the colors so I can make my instance look like my institution's theme?*. We show what changes you need to do in order to achieve that.
+
+We are going to change the top header section in the frontpage to apply our custom background color. Open the `assets/scss/<your instance shortname>/variables.scss` file and edit it as below:
+
+``` console
+$navbar_background_image: unset;
+$navbar_background_color: #000000; // Note this hex value is an example. Choose yours.
+```
+
+Then, run the `invenio-cli update --no-install-js` command as above and refresh the page! You should be able to see your top header's color changed! You can find all the available variables that you can change [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/scss/invenio_app_rdm/variables.scss).
+
+## Change search result
+
+Changing how your results are presented in the search page is also something quite common. We show how you can do that.
+
+We are going to update the search result template so we can show more text in the result's description. Create a file called `ResultsItemTemplate.jsx` and then edit it as below:
+
+```console
+import React from 'react';
+import {Item} from 'semantic-ui-react';
+import _truncate from 'lodash/truncate';
+
+export function ResultsItemTemplate(record, index) {
+  return (
+    <Item key={index} href={`/records/${record.id}`}>
+      <Item.Content>
+        <Item.Header>{record.metadata.titles[0].title}</Item.Header>
+        <Item.Description>
+          {_truncate(record.metadata.descriptions[0].description, { length: 400 })}
+        </Item.Description>
+      </Item.Content>
+    </Item>
+  )
+};
+```
+
+Then, run the `invenio-cli update --no-install-js` command as above and refresh the page! You should be able to see more text in each result's description! You can find all the available templates [here](https://github.com/inveniosoftware/invenio-app-rdm/tree/master/invenio_app_rdm/theme/assets/templates/search).
```

Comment:
doubt: `--no-install-js` is just for installing external, not building right, otherwise it is needed.

Reply:
By default it doesn't install, I just kept it for pointing out that there is this option. But I can omit it :)

Reply:
`--no-install-js` is just to skip installing the external npm packages. Here installing JS packages is not needed; updating the statics is the only thing needed.